.github/workflows/ci.yml

```diff
@@ -13,9 +13,20 @@ defaults:
     shell: "bash"
 
 env:
+  CI_PRERELEASE: "${{ github.event_name == 'push' }}"
   CI_RELEASE: "${{ github.event_name == 'release' }}"
   STACK_VERSION: "2.7.3"
 
+concurrency:
+  # We never want two releases or prereleases building at the same time, since
+  # they would likely both claim the same version number. Pull request builds
+  # can happen in parallel with anything else, since they don't mutate global
+  # state with a release. (GitHub Actions is either too cheap to give us `if`
+  # expressions or too lazy to document them, but we have untyped boolean
+  # operators to fall back on.)
+  group: "${{ github.event_name == 'pull_request' && github.run_id || 'continuous-deployment' }}"
```

Comment:
That comment about type conversion refers to equality comparisons, not boolean operators.

In practice, this pattern had the desired effect in testing; multiple pushes to master resulted in all but the last being canceled with the message `Canceling since a higher priority waiting request for 'continuous-deployment' exists`, but multiple PRs could still build concurrently.

Reply:
D'oh!